Day-I-Social-Card/social-card/src/App.css

```diff
@@ -2,3 +2,13 @@
   Write your styles/CSS for App.js here and import into the component that you need 
   IF you'd like to attempt to use less you'll have to research how to do this with Create React App
 */
+
+.App {
+  height: 500px;
+  width: 45%;
```

Comment:
`max-width: 500px; width: 100%` or something around that would have been appropriate. If the viewport width was short, then this component would only take up 45% of that space, i presume this number was arrived at with some bigger viewport in mind.